azurerm/resource_arm_traffic_manager_endpoint_test.go

```diff
@@ -666,6 +750,202 @@ resource "azurerm_traffic_manager_endpoint" "testExternalNew" {
 `, rInt, location, rInt, rInt, rInt, rInt)
 }
 
+func testAccAzureRMTrafficManagerEndpoint_subnets(rInt int, location string) string {
+	return fmt.Sprintf(`
+resource "azurerm_resource_group" "test" {
+  name     = "acctestRG-%d"
+  location = "%s"
+}
+
+resource "azurerm_traffic_manager_profile" "test" {
+  name                   = "acctesttmp%d"
+  resource_group_name    = "${azurerm_resource_group.test.name}"
+  traffic_routing_method = "Subnet"
+
+  dns_config {
+    relative_name = "acctesttmp%d"
+    ttl           = 30
+  }
+
+  monitor_config {
+    protocol = "https"
+    port     = 443
+    path     = "/"
+  }
+}
+
+resource "azurerm_traffic_manager_endpoint" "testExternal" {
+  name                = "acctestend-external%d"
+  type                = "externalEndpoints"
+  target              = "terraform.io"
+  profile_name        = "${azurerm_traffic_manager_profile.test.name}"
+  resource_group_name = "${azurerm_resource_group.test.name}"
+	subnet {
+		first = "1.2.3.0"
+		scope = "24"
+	}
+	subnet {
+		first = "11.12.13.14"
+    last = "11.12.13.14"
+	}
+}
+
+resource "azurerm_traffic_manager_endpoint" "testExternalNew" {
+  name                = "acctestend-external%d-2"
+  type                = "externalEndpoints"
+  target              = "www.terraform.io"
+  profile_name        = "${azurerm_traffic_manager_profile.test.name}"
+  resource_group_name = "${azurerm_resource_group.test.name}"
+	subnet {
```

Comment:
Could we use spaces instead of tabs here?

Reply:
Yes